File: ver_cpp/T03.cpp
// https://leetcode.cn/problems/sqrtx/
#include <bits/stdc++.h>
using namespace std;
class Solution
{
public:
	int mySqrt(int x)
	{
		return (int)sqrt(x);
	}
};

int main()
{
	int x;
	Solution s;
	cin >> x;
	cout << s.mySqrt(x) << endl;
	return 0;
}

File: ver_py/T03.py
# https://leetcode.cn/problems/sqrtx/
from math import sqrt
class Solution:
    def mySqrt(self, x: int) -> int:
        return int(sqrt(x))
    
if __name__ == "__main__":
    x = int(input())
    print(Solution().mySqrt(x))